atlp-rwanda/phantom-fe-elite
# #13 Feature: 404 page and Potecting routes

#### What does this PR do?

- Protecting routs from unauthorized users
- Redirecting non found pages to 404 page

#### Description of Task to be completed?

- Users must be authorized to access admins pages otherwise they will be redirected to the login page

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ft-404-page-13`

- Run `npm run start` to start the development server

- Navigate to admin route unauthorized, for example  `/updatedriver` check if you redirected to the login page

- Request non existing route and check if the `404 page` responded

#### What are the relevant Pivotal Tracker/Trello stories?

- https://trello.com/c/WfmuAzrs/13-404-page-and-potecting-routes

#### Screenshots (if appropriate)

-  N/A

#### Questions:

-  N/A
